lib/ApiRequestor.php

```diff
@@ -356,13 +387,25 @@ private function _requestRaw($method, $url, $params, $headers)
             $rawHeaders[] = $header . ': ' . $value;
         }
 
+        $requestStartMs = Util\Util::currentTimeMillis();
+
         list($rbody, $rcode, $rheaders) = $this->httpClient()->request(
             $method,
             $absUrl,
             $rawHeaders,
             $params,
             $hasFile
         );
+
+        try {
+            self::$requestTelemetry = new RequestTelemetry(
+                $rheaders['request-id'],
+                Util\Util::currentTimeMillis() - $requestStartMs
+            );
+        } catch (\Exception $e) {
```

Comment:
I was just being extra paranoid

Reply:
IMO we should still take this one out. Basically the list of things we're doing inside this `try` is: memory allocation, a call to `round`, a call to `microtime`, and subtraction. If any of those things happened to fail it seems like we'd have bigger problems than an uncaught exception.

Being conservative is generally good, but this sort of thing does bear some cost in that it can be somewhat mystifying to future maintainers who are trying to refactor things. Without the benefit of our current context, they'll tend to just cargo cult what's already in here, and every little thing adds a little bit to making changes more time consuming. So if we're ~certain that this is never going to be needed, I'd tend to leave it out.

Thoughts?

Reply:
Works for me. I just didn't want API calls to fail for some reason if metrics got screwy, but I see your point.  Incoming!